sirepo/package_data/static/js/srw.js

```diff
@@ -2596,6 +2596,16 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 return true;
             }
 
+            function logFileRequest(suffix) {
```

Comment:
This is a bit funny, because it is really not a "suffix" but a logical name. You could call it log_kind, as the function, and it would get passed to suffix.